examples/pug/index.js

```diff
@@ -5,6 +5,7 @@ const Hapi = require('hapi');
 const Vision = require('../..');
 const Path = require('path');
 const Pug = require('pug');
+const Path = require('path');
```

Comment:
Tests are failing because `Path` is already brought in 2 lines above =)

Reply:
Thanks, sorry about that, didn't realise it was already there.